docs/source/endpoints/comments.md

```diff
@@ -9,7 +9,12 @@ myst:
 
 # Comments
 
-Plone offers to users a feature to post comments on any content object with `plone.app.discussion`.
+```{note}
```

Comment:
I created a PR to unify instructions and which can be referenced here and in the `plone.restapi` upgrade guide, but it needs details.

https://github.com/plone/documentation/pull/1666

Let's change this entire `note` block with the following. Would you please also duplicate it in the [`plone.restapi` upgrade guide](https://6.docs.plone.org/plone.restapi/docs/source/upgrade-guide.html) as part of this PR?

````
```{versionchanged} 6.1
Discussion is disabled by default in Plone 6.1 and later.
To enable discussion, see the Plone 6.1 upgrade guide section {ref}`backend-upgrade-plone-v61-discussion-label`
```
````

Reply:
I think this is more for the Plone 6.1 upgrade guide, since it is not because of RestAPI. I have no good idea how to describe it there nor is it a breaking change for RestAPI. 
I would add a section to the https://github.com/plone/documentation/blob/6.0/docs/backend/upgrading/version-specific-migration/upgrade-to-61.md page.
Whats do you think?

Reply:
I was way ahead of you in https://github.com/plone/documentation/pull/1666/files 😁 

I think that references to the main docs are not required, but they would be a good thing to do. Imagine someone jumping into the REST API docs, and there is no note about this change. *sad trombone*